appinfo/app.php

```diff
@@ -38,13 +38,38 @@
 	\OC::$server->getLogger()->logException($e);
 	return;
 }
+
+\OC::$server->registerService('SAMLGroupDuplicateChecker', function() use($config) {
```

Comment:
```suggestion
\OC::$server->registerService(OCA\User_SAML\GroupDuplicateChecker::class, function() use($config) {
```

Using strings to register service is deprecated and we should migrate to use the class names